configs/aarch64.toml

```diff
@@ -0,0 +1,17 @@
+# The base address of the user space.
+user_space_base = 0x1000
```

Comment:
It not used? We still create `uspace` with the parameters hardcoded in `axmm`

Reply:
It will be used in the next PR, but here i will remove it.